aemsites/888de
# 163 link title fix

Fix 163

## Test URLs
- Before: https://main--888de--aemsites.hlx.page/
- After: https://163-link-title-fix--888de--aemsites.hlx.page/

## Testing instructions
Make sure links in footer don't have title attribute
Make sure no ill side effects
